src/__tests__/fromVariables.test.ts

```diff
@@ -56,6 +64,10 @@ describe('fromVariables', () => {
         symbol: 'T',
         type: 'DEPENDENT',
         dim: 4,
+        first: 3,
```

Comment:
What this change of first, last, min, max ?

Reply:
those properties have been included before this refactor, we should add those properties in the test case because the last version of jcampconverter returns it.